platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/virtualmerkle/map/smartcontracts/bytecode/SmartContractByteCodeMapKey.java

```diff
@@ -85,27 +88,37 @@ public void serialize(final SerializableDataOutputStream out) throws IOException
         out.writeLong(contractId);
     }
 
+    public void serialize(final WritableSequentialData out) {
+        out.writeLong(contractId);
+    }
+
     /**
      * {@inheritDoc}
      */
     @Override
-    public void deserialize(final SerializableDataInputStream in, final int version) throws IOException {
-        contractId = in.readLong();
+    @Deprecated(forRemoval = true)
```

Comment:
I removed all `forRemoval = true` for all methods deprecated as a part of this fix. Since these are not API methods provided by a library, but rather API interfaces to implement in an app, there is no question what to call instead. It's just "you now have to implement a few less methods" )

Reply:
All serialization methods that work with `ByteBuffer` can't be removed yet, they are still used by MerkleDb in JDB mode